recipes-ni/ni-hw-scripts/ni-hw-scripts-crio/init.d/handle_cpld_ip_reset

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# We are going to create a udev script to mount nirtfeatures under /dev
```

Comment:
We do have an udev rule to generate symlink under /dev/nirtfeatures, so maybe we can simplify this script.

```
(safemode) admin@NI-cRIO-9049-01CEEE05:~# cat /etc/udev/rules.d/localextra.rules
...
# create symlink to establish a uniform path to nirtfeatures
DRIVER=="nirtfeatures",RUN+="/bin/ln -sf /sys%p /dev/nirtfeatures"

#give all users read/write access to CPLD-created /dev/input/event?
SUBSYSTEM=="input", ENV{ID_PATH}=="acpi-NIC775D:??", KERNEL=="event?", MODE="0666"
```

However, it does mention in the comment that this can also handle case where the udev script doesn't run and then we can fall back. So we might want to keep the fallback logic.

Thoughts?

Reply:
We discussed in our standup today that if we have good testing, we can simplify the script. So I will go ahead and simplify the script. Then I will test it out on real hardware.